Serilog.Sinks.Network.Test/WhenLoggingViaTCP.cs

```diff
@@ -59,11 +51,11 @@ public void CanLogHelloWorld_WithDefaultFormatter()
         [Fact]
         public void CanLogHelloWorld_WithRawFormatter()
         {
-            ConfigureTestLogger(new RawFormatter());
-            var arbitraryMessage = nameof(WhenLoggingViaTCP) + "CanLogHelloWorld_WithRawFormatter" + Guid.NewGuid();
+            ConfigureTestLogger(new CompactJsonFormatter());
```

Comment:
same question (from ignorance on my part here) does this change the format of the message significantly?

Reply:
It does change things a bit, but it still has no impact on the end-users. The `RawFormatter` is deprecated, so I replaced it. However, end-users can continue to use `RawFormatter` if they prefer. In this test case, the output is slightly different, but overall, I believe everything still works as expected.